libparsec/crates/client/src/workspace/transactions/move_entry.rs

```diff
@@ -386,42 +386,36 @@ async fn move_entry_different_parents<'a>(
 
     let now = ops.device.time_provider.now();
     let src_parent_id = updater.src_parent_manifest.base.id;
-    let src_child_id = updater.src_child_manifest.id();
+    let child_id = updater.src_child_manifest.id();
     let dst_parent_id = updater.dst_parent_manifest.base.id;
 
     let mut_src_parent = Arc::make_mut(&mut updater.src_parent_manifest);
-    let child_id = mut_src_parent
-        .children
-        .remove(&src_child_name)
-        .expect("already checked");
-    mut_src_parent.updated = now;
-    mut_src_parent.need_sync = true;
+    // Remove child from source parent
+    let mut src_parent_data = HashMap::from_iter([(src_child_name.clone(), None)]);
```

Comment:
Same comment as in https://github.com/Scille/parsec-cloud/pull/8294, maybe add:
```rust
dst_parent_data.insert(dst_child_name, Some(child_id));
```

Reply:
done